pkg/volume/cinder/attacher.go

```diff
@@ -293,10 +329,14 @@ func (detacher *cinderDiskDetacher) Detach(deviceMountPath string, nodeName type
 	}
 
 	if err = detacher.cinderProvider.DetachDisk(instanceID, volumeID); err != nil {
```

Comment:
Yes,  the cinder provider can't handle duplicate detach, we need to check if operation is pending. I update it soon.